app/views/drivers/_form.html.erb

```diff
@@ -0,0 +1,12 @@
+<%# in app/views/drivers/_form.html.erb %>
+<h2> Form page </h2>
+<%= form_with model: @driver, class: 'create-driver' do |f| %>
+  <p>Please provide the following information :</p>
+  <%= f.label :name %>
+  <%= f.text_field :name %>
+  <%= f.label :vin%>
+  <%= f.text_field :vin %>
+   <%= f.label :available %>
+  <%= f.text_field :available%>
```

Comment:
This is a little awkward as a text field. There's some implicit conversion the ruby code is doing when it sends it along to the model. If a user enters "no" into that field, it will get saved as true.